cypress/e2e/Counter.cy.tsx

```diff
@@ -0,0 +1,13 @@
+describe('Counter', () => {
+    it('should increment the counter', () => {
+        cy.visit('http://localhost:3000')
```

Comment:
It seems like typescript doesn't know about Cypress, so it gives the warning "Cannot find name 'cy'." 
You can find the solution here: https://docs.cypress.io/guides/tooling/typescript-support